core/src/components/item/item.scss

```diff
@@ -523,6 +538,19 @@ ion-ripple-effect {
   z-index: 1;
 }
 
+// Item Max Length Counter
+// --------------------------------------------------
+
+.item-counter {
+  @include margin-horizontal(auto, null);
+
+  padding-inline-start: 16px;
+
+  color: var(--counter-color);
```

Comment:
Same thing here. Is it possible to just fix the issue in https://github.com/ionic-team/ionic-framework/issues/23892 and create a ticket to add this var later?